### PR TITLE
Add syntax highlighting via highlightjs

### PR DESCRIPTION
**Pull request by pentaphobe**
- add a packed [highlightjs](https://highlightjs.org/), and its default theme CSS
  - this is a custom build of highlightjs with only a few languages included to keep file size down:
    - Go, Java, Bash, "shell sessions" (presumedly interactive session dumps), and Markdown (just in case)
- add a single CSS style override to reset highlightjs' default padding (this ruined a couple layouts)
- update root slides template:
  - include aforementioned JS+CSS files
  - add short script:
    - bootstraps highlightjs styling on go-slides' code blocks
    - configures highlightjs language detection in a specific order to avoid overzealous detection of shell commands as golang
- add 3rd party license file
  - currently only contains LICENSE from highlightjs package

NB: The default language autodetection was oddly picking up some of the `shell` session code blocks as `golang`, and the `Java` as `CSS`, hence the additional config override.

**Feel free to disregard this PR - just busywork while watching a film**

### CODE REVIEW

**Comment by pentaphobe** [comment]
just realised my new machine wasn't set up to sign commits, can re-push if verified commits are required.

File: LICENSE-3RD-PARTY [hunk 1]
```diff
@@ -0,0 +1,29 @@
+-----------------------------------------------------------------------------
```
Comment (anzdaddy) [inline LICENSE-3RD-PARTY:1]: [linguist-vendored](https://github.com/github/linguist#vendored-code) as appropriate

**Review by anzdaddy — APPROVED** [review]

File: LICENSE-3RD-PARTY (hunk 1 above)
Comment (juliaogris) [reply LICENSE-3RD-PARTY:1]: Wasn't it @pentaphobe who introduced us to linguist-vendored in the first place??

**Review by juliaogris — APPROVED** [review]
This is super cool!!! THANK YOU! 
Does it actually work? have you tried it?
I just made you editor of the `gotraining` gcloud with your `pentaphobe@gmail.com` account.
You could just push it:
`gcloud app deploy` and then view it at http://gotraining.appspot.com
Alternatively if you have go installed on your local machine you could try it locally as:
```
go build .
./go-slides -base . -content content
```
If it works: please merge. CHAMPION!!!! (you should watch films more often.)

File: LICENSE-3RD-PARTY (hunk 1 above)
Comment (pentaphobe) [reply LICENSE-3RD-PARTY:1]: :giggle:

so I've really no excuse for that dangling CSS file...

The third party license file is at least partially manual

**Comment by pentaphobe** [comment]
> Does it actually work? have you tried it?

Sure have - I'm not confident enough to push code I haven't run, which is just as well as there were a few oddities with the default language detection and style overrides

I ran it locally with just:

```
./go-slides -base .
```


(psst. why do I need to watch films more often?)

**Pull request merged**